__tests__/app.test.js

```diff
@@ -140,4 +116,60 @@ describe('GET /api/articles/:article_id', () => {
         .then((res) => {
             expect(res.body).toEqual({message: 'Bad Request'})
         })
-    });
\ No newline at end of file
+    });
+});
+describe('GET /api/articles/:article_id/comments', () => {
+    test('should return 200 OK status and an array of all comments for a specific article, sorted in descending order by created_at', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body).toBeSorted({ descending: true, key: 'created_at' })
```

Comment:
body itself should not be the array of comments, body should have a key of comments with the array of comments as its value.